bigquery/routine_integration_test.go

```diff
@@ -121,7 +123,14 @@ func TestIntegration_RoutineRemoteUDF(t *testing.T) {
 			TypeKind: "STRING",
 		},
 	}
-	if err := routine.Create(ctx, meta); err != nil {
+
+	err = internal.Retry(ctx, gax.Backoff{}, func() (stop bool, err error) {
+		if err := routine.Create(ctx, meta); err != nil {
+			return false, err
```

Comment:
Do you want to evaluate this via `retryableError()` or similar?

Reply:
do you feel like is needed as this is just for integration test ?

Reply:
We can leave as is and swap if it causes issues down the line.